axonserver/src/main/java/io/axoniq/axonserver/access/ApplicationBinding.java

```diff
@@ -0,0 +1,14 @@
+package io.axoniq.axonserver.access;
+
+public class ApplicationBinding {
+
+    private final String name;
+
+    public ApplicationBinding(String name) {
+        this.name = name;
+    }
+
+    public String name() {
+        return name;
+    }
+}
```

Comment:
Can you add (at least) a `toString()` so logging will show the type of Binding (Application) as well as the name?